code/datums/spells/banana_touch.dm

```diff
@@ -44,15 +44,25 @@
 		unEquip(wear_mask, TRUE)
 		unEquip(head, TRUE)
 		unEquip(wear_suit, TRUE)
-		equip_to_slot_if_possible(new /obj/item/clothing/head/wizard/clown, slot_head, TRUE, TRUE)
+		if(src.dna.species.name == "Plasmaman")
```

Comment:
Come on dude, you've been around long enough to know that `src` is always implied.

Reply:
Lmao
I'm not familiar with whatever that second thing is. Mind pointing me towards the proc name and I'll edit it in there?

Reply:
Guessing that what Fox is suggesting is something like:
if(is_species(src, /datum/species/plasmaman))
https://github.com/ParadiseSS13/Paradise/blob/master/code/__HELPERS/mobs.dm#L460

Reply:
Good call; it's changed. Thank you guys.